packages/devtools_app/lib/src/screens/deep_link_validation/validation_details_view.dart

```diff
@@ -83,23 +84,27 @@ class ValidationDetailHeader extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return OutlineDecoration(
-      child: Padding(
-        padding: const EdgeInsets.symmetric(horizontal: defaultSpacing),
-        child: Row(
-          mainAxisAlignment: MainAxisAlignment.spaceBetween,
-          children: [
-            Text(
-              viewType == TableViewType.domainView
-                  ? 'Selected domain validation details'
-                  : 'Selected Deep link validation details',
-              style: Theme.of(context).textTheme.titleSmall,
-            ),
-            IconButton(
-              onPressed: () =>
-                  controller.updateDisplayOptions(showSplitScreen: false),
-              icon: const Icon(Icons.close),
-            ),
-          ],
+      showLeft: false,
+      child: SizedBox(
+        height: actionWidgetSize,
+        child: Padding(
```

Comment:
use a Container widget instead, where you can set both `height1 and `padding` attributes